plugins/rapture/nexus-rapture-baseapp/src/main/baseapp/packages/rapture-theme/sass/src/Ext/button/Button.scss

```diff
@@ -92,12 +92,12 @@
   }
 }
 
-.feature-content .x-panel-header .x-btn-drilldown-medium {
+.nx-feature-content .x-panel-header .x-btn-nx-drilldown-medium {
```

Comment:
Not sure why we would deviate from prefixing with nx to embedding in the middle... is this an ExtJS thing?